src/puya/awst_build/intrinsic_data.py

```diff
@@ -392,6 +398,7 @@
                     allowed_types=[
                         wtypes.bytes_wtype,
                         bytes,
+                        wtypes.account_wtype,
```

Comment:
hmmm, assuming we stick with targeting multiple versions as a CLI option, how do we gate stuff like this?